libmscore/layout.cpp

```diff
@@ -3882,6 +3894,7 @@ System* Score::collectSystem(LayoutContext& lc)
                               if (!s->enabled())
                                     s->setEnabled(true);
                               }
+                        // TODO: use findPotentialSectionBreak here to handle breaks on frames correctly?
```

Comment:
Note to self: yes, I was able to break this setting a section break on a vertical frame then editing something on the following section, the measure number reappeared at the start of the system.